.github/workflows/ci-lambda.yml

```diff
@@ -0,0 +1,59 @@
+name: CI - Python
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+    paths:
+      - '**/*.py'
+      - '**/*requirement*.txt'
+
+  merge_group:
```

Comment:
Do we need this line?

Reply:
Oh nevermind, I see it in our existing config

Reply:
per https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#merge_group

If your repository uses GitHub Actions to perform required checks on pull requests in your repository, you need to update the workflows to include the merge_group event as an additional trigger. Otherwise, status checks will not be triggered when you add a pull request to a merge queue. The merge will fail as the required status check will not be reported. The merge_group event is separate from the pull_request and push events.